temporalcli/commands.workflow.go

```diff
@@ -41,3 +95,77 @@ func (*TemporalWorkflowTraceCommand) run(*CommandContext, []string) error {
 func (*TemporalWorkflowUpdateCommand) run(*CommandContext, []string) error {
 	return fmt.Errorf("TODO")
 }
+
```

Comment:
Although it's a private function, it would be good to document this something like
> This function guarantees that exactly one of the 3 returned values will be non-nil.

Reply:
Concur

Reply:
OK great, I can make these minor changes.